net/Socket.cpp

```diff
@@ -1087,21 +1087,21 @@ std::shared_ptr<Socket> ServerSocket::accept()
             const void *inAddr;
             if (clientInfo.sin6_family == AF_INET)
             {
-                auto ipv4 = (struct sockaddr_in *)&clientInfo;
+                struct sockaddr_in *ipv4 = (struct sockaddr_in *)&clientInfo;
```

Comment:
we could probably replace the c-style "struct sockaddrWhatever" with c++-style "sockaddrWhatever", but that's just an observation.